contrib/ci/check-source.py

```diff
@@ -212,6 +212,7 @@ def _test_line_blocked_fns(self, line: str) -> None:
             "&= ~(1ull <<": "Use FU_BIT_CLEAR() instead",
             "__attribute__((packed))": "Use rustgen instead",
             "memcpy(": "Use fu_memcpy_safe or rustgen instead",
+            " ioctl(": "Use fu_udev_device_ioctl() instead",
```

Comment:
drop the leading space for the comment

Reply:
Nah, that's load bearing. Otherwise we match fu_udev_device_**ioctl(**

Reply:
Oh wow; OK.